docs/quickstarts/openshift-ai-rhods-fasttrack/metadata.yml

```diff
@@ -0,0 +1,6 @@
+kind: QuickStarts
+name: openshift-ai-rhods-fasttrack
+tags:
+  - kind: bundle
+    value: openshift-ai
```

Comment:
Currently, there is no `openshift-ai` bundle in HCC. The bundles are named based on the URL fragment.

Where in HCC do you expect the tile should be visible?